Zilon.Core/Zilon.Core/ProgressStoring/HumanPlayerStorageData.cs

```diff
@@ -1,40 +1,23 @@
-﻿using System.Linq;
-
-using Zilon.Core.Players;
-using Zilon.Core.World;
+﻿using Zilon.Core.Players;
 
 namespace Zilon.Core.ProgressStoring
 {
     public sealed class HumanPlayerStorageData
     {
-        public string SectorSid { get; set; }
-
-        public int CurrentGlobeNodeX { get; set; }
-        public int CurrentGlobeNodeY { get; set; }
-
-        public int TerrainX { get; set; }
-        public int TerrainY { get; set; }
-
         public static HumanPlayerStorageData Create(HumanPlayer humanPlayer)
         {
+            if (humanPlayer is null)
+            {
+                throw new System.ArgumentNullException(nameof(humanPlayer));
+            }
+
             var storageData = new HumanPlayerStorageData();
-            storageData.CurrentGlobeNodeX = humanPlayer.GlobeNode.OffsetX;
-            storageData.CurrentGlobeNodeY = humanPlayer.GlobeNode.OffsetY;
-            storageData.SectorSid = humanPlayer.SectorSid;
-            storageData.TerrainX = humanPlayer.Terrain.Coords.X;
-            storageData.TerrainY = humanPlayer.Terrain.Coords.Y;
             return storageData;
         }
 
-        public void Restore(HumanPlayer humanPlayer, Globe globe, IWorldManager worldManager)
+        public void Restore(HumanPlayer humanPlayer)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)